[0.000000] (-) TimerEvent: {}
[0.000450] (-) JobUnselected: {'identifier': 'dolly'}
[0.000645] (-) JobUnselected: {'identifier': 'dolly_follow'}
[0.001090] (-) JobUnselected: {'identifier': 'dolly_gazebo'}
[0.001137] (-) JobUnselected: {'identifier': 'dolly_tests'}
[0.001465] (-) JobUnselected: {'identifier': 'more_interfaces'}
[0.001496] (-) JobUnselected: {'identifier': 'py_pubsub'}
[0.001517] (-) JobUnselected: {'identifier': 'py_srvcli'}
[0.001534] (-) JobUnselected: {'identifier': 'python_parameters'}
[0.001704] (-) JobUnselected: {'identifier': 'turtlesim'}
[0.001742] (-) JobUnselected: {'identifier': 'tutorial_interfaces'}
[0.001773] (my_package) JobQueued: {'identifier': 'my_package', 'dependencies': OrderedDict()}
[0.001799] (my_package) JobStarted: {'identifier': 'my_package'}
[0.100001] (-) TimerEvent: {}
[0.200657] (-) TimerEvent: {}
[0.301371] (-) TimerEvent: {}
[0.402337] (-) TimerEvent: {}
[0.502610] (-) TimerEvent: {}
[0.603096] (-) TimerEvent: {}
[0.704437] (-) TimerEvent: {}
[0.791095] (my_package) StderrLine: {'line': b'Traceback (most recent call last):\n  File "/usr/lib/python3/dist-packages/colcon_core/executor/__init__.py", line 91, in __call__\n    rc = await self.task(*args, **kwargs)\n  File "/usr/lib/python3/dist-packages/colcon_core/task/__init__.py", line 93, in __call__\n    return await task_method(*args, **kwargs)\n  File "/usr/lib/python3/dist-packages/colcon_ros/task/ament_python/build.py", line 51, in build\n    setup_py_data = get_setup_data(self.context.pkg, env)\n  File "/usr/lib/python3/dist-packages/colcon_core/task/python/__init__.py", line 20, in get_setup_data\n    return dict(pkg.metadata[key](env))\n  File "/usr/lib/python3/dist-packages/colcon_ros/package_augmentation/ros_ament_python.py", line 57, in getter\n    return get_setup_information(\n  File "/usr/lib/python3/dist-packages/colcon_python_setup_py/package_identification/python_setup_py.py", line 241, in get_setup_information\n    _setup_information_cache[hashable_env] = _get_setup_information(\n  File "/usr/lib/python3/dist-packages/colcon_python_setup_py/package_identification/python_setup_py.py", line 281, in _get_setup_information\n    result = subprocess.run(\n  File "/usr/lib/python3.8/subprocess.py", line 516, in run\n    raise CalledProcessError(retcode, process.args,\nsubprocess.CalledProcessError: Command \'[\'/usr/bin/python3\', \'-c\', "import sys;from setuptools.extern.packaging.specifiers import SpecifierSet;from distutils.core import run_setup;dist = run_setup(    \'setup.py\', script_args=(\'--dry-run\',), stop_after=\'config\');skip_keys = (\'cmdclass\', \'distclass\', \'ext_modules\', \'metadata\');data = {    key: value for key, value in dist.__dict__.items()     if (        not key.startswith(\'_\') and         not callable(value) and         key not in skip_keys and         key not in dist.display_option_names    )};data[\'metadata\'] = {    k: v for k, v in dist.metadata.__dict__.items()     if k not in (\'license_files\', \'provides_extras\')};sys.stdout.buffer.write(repr(data).encode(\'utf-8\'))"]\' returned non-zero exit status 1.\n'}
[0.791774] (my_package) JobEnded: {'identifier': 'my_package', 'rc': 1}
[0.801761] (-) EventReactorShutdown: {}
